twine-upload.sh

```diff
@@ -28,6 +28,10 @@ then
         are in place should you face this problem.
 fi
 
+if [[ ${INPUT_CHECK,,} != "false" ]] ; then
+  exec twine check ${INPUT_PACKAGES_DIR%%/}/*
```

Comment:
`exec` replaces the current process and so when used, it'll effectively prevent upload from running:
```suggestion
  twine check ${INPUT_PACKAGES_DIR%%/}/*
```